packages/public-api/tsconfig.build.json

```diff
@@ -2,7 +2,10 @@
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "./build",
-    "module": "commonjs"
+    "module": "commonjs",
+    "paths": {
+      "@gradejs-public/shared": ["../shared/src"]
```

Comment:
It'll probably break the deploy logic. We should use path aliases only in the development mode (specified in `tsconfig.json` in the root). Production configs should not use path aliases for monorepo packages